### PR TITLE
README: added instructions to use a different port if needed

### PR DESCRIPTION
**Pull request by rncrtr**

### CODE REVIEW

**Comment by rjVapes** [comment]
@rncrtr Can you resolve the conflicts, and we can get this in.  Port can also be set via config if you want to mention that, and on windows with a SET, etc.  Feel free to add all that but even what you've already added is more documentation, and hard to say no to that :+1:

**Comment by rncrtr** [comment]
Sorry, I am not seeing conflicts? Should I be merging to a different branch?

**Comment by rjVapes** [comment]
Not sure, it's not currently mergable due to conflict.  Most likely the random whitespace edits conflict with some things people have changed in the last few months.

**Comment by rncrtr** [comment]
Ok, this should be good to go. I figured it out. Cool.

File: README.md [hunk 1]
```diff
@@ -168,9 +179,15 @@ $ mean help [command]
   <p>For example, the <i>admin</i> role is required to edit tokens.</p>
 
 ```bash
+<<<<<<< HEAD
```
Comment (rjVapes) [inline README.md]: Now you have a bunch of merge stuff in the file like this.

**Comment by rncrtr** [comment]
Ok, so sorry. I am still learning some of the git stuff. I think i have it fixed again. Thanks for your patience.

**Pull request merged**